src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -791,6 +791,14 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             if (!should_fuse)
                 return;
 
+            if (_lo.get_optimization_attributes().use_onednn_impls
+                && input.is_type<reorder>() && input.get_users().size() == 1
+                && input.get_input_layouts()[0].format == input.get_output_layout().format
+                && input.get_input_layouts()[0].data_type != input.get_output_layout().data_type
+                && activation_node.get_primitive()->activation_function == activation_func::swish) {
+                return;
+            }
```

Comment:
I think we need to be more loose here:
```
            // Onednn reorder does not support eltwise nor binary post operation
            if (_lo.get_optimization_attributes().use_onednn_impls && input.is_type<reorder>()) {
                return;
            }
```

Reply:
did you check the graph impact with dry-run?

Reply:
No issue.